docs/tutorials/ui-customization.md

```diff
@@ -55,6 +55,7 @@ The following elements can be added to the UI bar using this configuration value
 * fullscreen: adds a button that toggles full screen mode on click.
 * overflow_menu: adds a button that opens an overflow menu with additional settings
   buttons. It's content is also configurable.
+* context_menu: adds a custom context menu on right-click.
```

Comment:
Design-wise, I don't think this belongs inside the `controlPanelElements`. Up until now, every element that can be added to `controlPanelElements`  has been a button that physically appears in the space above the seek bar. The play button, the rewind button, the button that opens the overflow menu, and so on.
Things that do not appear in that space are all boolean configuration values; e.g. `addSeekBar`, `addBigPlayButton`.
I think this would be better off as another boolean configuration value. Something like `rightClickForContextMenu` (riffing on the existing `doubleClickForFullscreen`).

Reply:
Thank you Theodab! I agree with the new design and have performed the configuration changes. I decided to go with `customContextMenu` as it can also be present in mobile devices through a long tap instead of a right-click!

I followed the same structure of other boolean UI elements (like `addBigPlayButton` or `addBigPlayButton`), but I am not very familiar with custom configurations inside `ui/controls.js` and would really appreciate a follow up review on that aspect, now that the context menu is out of the regular `controlPanelElements` configuration loop.